flink-table/flink-table-common/src/main/java/org/apache/flink/table/connector/ChangelogMode.java

```diff
@@ -149,7 +149,7 @@ public String toString() {
             return kinds.toString();
         } else {
             return kinds.stream()
-                    .map(kind -> kind == RowKind.DELETE ? "~D" : kind.toString())
+                    .map(kind -> kind == RowKind.DELETE ? "~DELETE" : kind.toString())
```

Comment:
nit: The change is fine - I wondered what you thought of having the toString as `~DELETE (Partial Delete)`, just to aid readability in the `toString`.